index.html

```diff
@@ -0,0 +1,133 @@
+            While music is an important part of Sziget, art is also a core element. There are multiple venues including a circus, theater, museum quarter and much more. Artists and performers come 
+            from all over the world to display their talents. There are also opportunities for Szitizens and artists to work together in order to create memorable and meaningful artwork.
+          </p>
+        </div>
+        <div class="card-b col-12-xs col-12-sm col-5-md col-3-lg col-2-xl">
+          <h3 class="card-title">
+            Wherever you go
+          </h3>
+          <p class="text-green-light-6">
+            Roads give you chances to seek and explore. Want an adventure? Just open your door. Award-winning author Pat Zietlow Miller's lilting rhymes celebrate the possibilities that 
+            lie beyond the next bend in the road--the same road that will always lead you home again. Look back at Sziget 2019, where these words were brought to life on the Island of Freedom!
+          </p>
+        </div>
+        <div class="card-b col-12-xs col-12-sm col-5-md col-3-lg col-2-xl">
+          <h3 class="card-title">
+            Join the Love Revolution
+          </h3>
+          <p class="text-green-light-6">
+            We believe that the power of diversity is a unifying force that can help create change around us. We come in all shades, from different backgrounds, 
+            but no matter what we are a community who share common values.
+          </p>
+        </div>
+        <div class="card-b col-12-xs col-12-sm col-5-md col-3-lg col-2-xl">
+          <h3 class="card-title">
+            Our cause: SUPERAR
+          </h3>
+          <p class="text-green-light-6" id="ticketslink">
+            Sziget partnered with Superar this year, to support the program’s initiative on spreading music to children coming from underprivileged backgrounds. 
+            Music is a powerful tool that can help children improve their focus, motivation, creativity and confidence, while allowing them to overcome socio-economic boundaries. 
+            Every reusable cup dropped off at a collection point at Sziget, went towards supporting the Superar program.
+          </p>
+        </div>
+      </div>
+      <button type="button" class="btn-orange btn-or-b mt-2" id="featured">Buy your ticket HERE!</button>
+    </div>
+    <div class="bg-yellow-light-9 featured-artists pb-2">
+      <h2 class="h-secondary pt-2">Featured Artists</h2>
+      <div class="separator"></div>
+      <div class="row" id="artistsholder"></div>
+    </div>
+  <footer class="bg-gray-dark-7 text-white">
+    <div class="row gap-2 footer">
+      <div>
+        <p class="text-footer">
+          copyright 2022 SZIGET festival <br>
+          All Rights reserved
+        </p> 
+      </div>
+      <div>
+        <img class="footer-img" src="./images/sziget-logo-footer-removebg-preview.png" alt="sziget logo">
+      </div>
+    </div>
+  </footer>
```

Comment:
- [ ] Please add the partner's section as seen in the original design.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/190464879-c38cd461-46d7-4b7c-a1f9-68a98152508b.png)
### Required
![image](https://user-images.githubusercontent.com/45460192/190465293-0d8e5202-b842-44da-bc05-e881910789fa.png)

Reply:
Partners are in the 'about' page